geps/gep-1713/index.md

```diff
@@ -438,35 +446,38 @@ spec:
     sectionName: second
 ```
 
-For instance, the following `HTTPRoute` attempts to attach to a listener defined in the parent `Gateway` using the sectionName `foo`. This is not valid and the route's status `Accepted` condition should be set to `False`
-
+To attach to listeners in both a `Gateway` and `ListenerSet` the route MUST have two `parentRefs`:
 ```yaml
 apiVersion: gateway.networking.k8s.io/v1
 kind: HTTPRoute
 metadata:
   name: httproute-example
 spec:
   parentRefs:
-  - name: some-workload-listeners
+  - name: second-workload-listeners
     kind: ListenerSet
+    sectionName: second
+  - name: parent-gateway
+    kind: Gateway
     sectionName: foo
 ```
 
-To attach to listeners in both a `Gateway` and `ListenerSet` the route MUST have two `parentRefs`:
+For instance, the following `HTTPRoute` attempts to attach to a listener defined in the parent `Gateway` using the sectionName `foo`. This is not valid and the route's status `Accepted` condition should be set to `False`
+
 ```yaml
 apiVersion: gateway.networking.k8s.io/v1
 kind: HTTPRoute
 metadata:
   name: httproute-example
 spec:
   parentRefs:
-  - name: second-workload-listeners
-    kind: ListenerSet
-    sectionName: second
-  - name: parent-gateway
+  - name: some-workload-listeners
```

Comment:
I guess no real change here, just order, right?

Reply:
right, I wrote on my self review why I did the change. The example of the attachment of Gateway AND ListenerSet is more clear on this order, then we can go to the invalid examples :)

Reply:
To make this invalid the `kind` needs to be a `ListenerSet` in the parentRef